.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
```

Comment:
I'm not clear -- what does package-ecosystem `docker` do here?

Reply:
Should we remove it then?

Reply:
done